### PR TITLE
Remove deprecated plugins.sh and add warning to install-plugins.sh

### PR DESCRIPTION
**Pull request by slide**
This removes the old plugins.sh that has been deprecated for some time. It also adds a warning to install-plugins.sh mentioning that is is deprecated.

### CODE REVIEW

File: install-plugins.sh [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
 set -o pipefail
 
+echo "WARN: install-plugins.sh is deprecated, please switch to jenkins-plugin-cli.sh"
```
Comment (timja) [inline install-plugins.sh:23]:
```suggestion
echo "WARN: install-plugins.sh is deprecated, please switch to jenkins-plugin-cli"
```

Comment (timja) [reply install-plugins.sh:23]: should we add this to the bottom too?

Comment (slide) [reply install-plugins.sh]: We only had it at the top for plugins.sh I believe, so I don't think its needed.

**Comment by slide** [comment]
Any issues with this?

**Review by MarkEWaite — APPROVED** [review]

**Pull request merged**